sdk/storage/azblob/highlevel.go

```diff
@@ -59,17 +59,17 @@ type HighLevelUploadToBlockBlobOption struct {
 	TransactionalContentMD5 *[]byte
 }
 
-func (o *HighLevelUploadToBlockBlobOption) getStageBlockOptions() *StageBlockOptions {
-	leaseAccessConditions, _ := o.BlobAccessConditions.pointers()
-	return &StageBlockOptions{
+func (o *HighLevelUploadToBlockBlobOption) getStageBlockOptions() *BlockBlobStageBlockOptions {
```

Comment:
If this is a renaming (`StageBlockOptions` to `BlockBlobStageBlockOptions`), this should be in the changelog.

Reply:
There are a lot of such changes. It'll not be feasible to list them all. So, I've added one liner in changelog to address this.